pkg/provisioner/resourcehandlers.go

```diff
@@ -53,6 +53,10 @@ const (
 	objectBucketNameFormat = "obc-%s-%s"
 )
 
+func configMapNameForClaim(obc *v1alpha1.ObjectBucketClaim) string {
+	return obc.Name
+}
+
```

Comment:
Sorry I didn't understand use case of this function, we can directly access obc.Name in all its occurences. Please ignore it if is lack of my golang style of coding

Reply:
I intend this just to make it a little clearer that the claim informs the name of the configmap and allows for it to change more easily in the future now that it is used in more than one place in the code.